cognite_toolkit/_cdf_tk/utils/interactive_select.py

```diff
@@ -157,6 +158,45 @@ def _get_aggregator(self, client: ToolkitClient) -> AssetCentricAggregator:
         return EventAggregator(self.client)
 
 
+class RawTableInteractiveSelect:
+    def __init__(self, client: ToolkitClient, operation: str) -> None:
+        self.client = client
+        self.operation = operation
+
+    def _available_databases(self) -> list[str]:
+        databases = self.client.raw.databases.list(limit=-1)
+        return [db.name for db in databases if db.name is not None]
```

Comment:
Question, not blocking: Would it make sense to make it possible to select unnamed databases as well, or at least indicate their existence so people can fix the missing name?

Reply:
There are no unnamed databases in CDF. This is simply to satisfy the type checker, due to a weakness in the py-sdk.